### DISCUSSION TITLE
Discerning rule(s) being autofixed

### DISCUSSION THREAD

**Discussion by jamesbraza** [astral-sh/ruff#4319] [Q&A] [answered]
Running this: `ruff --fix --select=ALL a.py`, ruff autofixes some stuff, and prints non-fixable remaining codes.

Question: how can one get `ruff` to print what rule codes were autofixed?

I am not sure what rule is generating an autofix, and I tried `--verbose` flag but it doesn't detail what was fixed.

**Comment by charliermarsh** [comment, marked as answer]
Can you try `--show-fixes`?

**Reply by jamesbraza** [reply]
Thank you, apparently I missed that in my reading of the docs.

Perhaps consider `--verbose` enabling `--show-fixes`
